### DISCUSSION TITLE
Is there an easy way to reinstall all apps (I have not installed via Home Brew in the first place) via Home Brew (if available if course)?

### DISCUSSION THREAD

**Discussion by qns7** [Homebrew/discussions#5410] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9fdb0d53990d3e62e25325a6f89dbb769e84a530
Last commit: 18 hours ago
Core tap JSON: 27 May 18:37 UTC
Core cask tap JSON: 27 May 18:37 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.1 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.1/bin/ruby
CPU: octa-core 64-bit arm_ibiza
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

The title says it all basically. 
It would be cool if there exists an easy way without having to deinstall every app manually via AppCleaner and then reinstall it via Home Brew. Also: I don't know if my settings/configs/etc are gone by doing it the manual way(?). 
Why: It would be convinient to have the most apps in one place (Home Brew) which is so easy to maintain. I'm new to macOS and Home Brew so please be friendly if I'm asking a dumb question here.^^
Best regards Q

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Tips-N'-Tricks#appoint-homebrew-cask-to-manage-a-manually-installed-app

**Reply by qns7** [reply]
> https://docs.brew.sh/Tips-N'-Tricks#appoint-homebrew-cask-to-manage-a-manually-installed-app

Awesome! Thank you very much and sorry for obviously not searching the Doc properly.
Best regards Q
